config/cloud.cfg.tmpl

```diff
@@ -66,6 +70,14 @@ network:
   config: disabled
 {% endif %}
 
+{% if variant == "rhel" %}
+# Default redhat settings:
+ssh_deletekeys:   1
```

Comment:
Please make this `ssh_deletekeys: true` instead of 1. As we are trying to make cloud-init-schema.json more strict instead of permissive on different value types. Tip of master will currently emit a warning in logs (and eventually will error completely) with this log `Cloud config schema errors: ssh_deletekeys: 1 is not of type 'boolean'
`